backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlSessionFieldFactory.java

```diff
@@ -30,10 +31,13 @@ public GraphQLFieldDefinition signoutField(Database database) {
         .type(GraphqlApiMutationResult.typeForMutationResult)
         .dataFetcher(
             dataFetchingEnvironment -> {
-              String user = database.getActiveUser();
-              database.setActiveUser(GraphqlConstants.ANONYMOUS);
+              MolgenisSessionManager sessionManager =
+                  dataFetchingEnvironment.getGraphQlContext().get(MolgenisSessionManager.class);
```

Comment:
getGraphQLContext allows us to pass a wrapper to our session manager; we implement this MolgenisSessionManager in the *.web module